tests/publiccloud/storage_perf.pm

```diff
@@ -161,11 +168,11 @@ sub run {
         # Store values in influx-db
         if ($url) {
             my $data = {
-                table => 'storage',
+                table => get_var('PUBLIC_CLOUD_FIO_TABLE', 'storage_fio'),
```

Comment:
I don't think this is something which suppose to be dynamic . Such things suppose to be hardlinked to certain test and I don't see why do you want to have possibility to have same test type to write into different tables

Reply:
The idea comes noting that we already have parameters for db `PUBLIC_CLOUD_PERF_DB` and even url `PUBLIC_CLOUD_PERF_DB_URI`, rarely changing, but table don't, useful instead to address temporary VR tests data separated from production environment.

Reply:
Anyway, I will restore the hardlink

Reply:
I set the table name: `storage_fio`